src/libmongoc/doc/static/styles.css

```diff
@@ -107,4 +107,14 @@ aside.footnote> :not(:first-child) {
 /* Pygments does not set an .nv color for body[data-theme="light"], which makes them a bit indistinguishable. */
 body[data-theme="light"] .highlight .nv {
   color: #0004ff;
+}
+
+a.reference.external {
+  text-decoration: underline;
+}
+
```

Comment:
```suggestion
```

Suggest removing the underline since we are already using `⇗` to mark external links. It makes some text ugly and difficult to read.

![image](https://github.com/mongodb/mongo-c-driver/assets/88335979/1d616577-58c4-45ff-98a4-5962d10fd9ba)